docs/Manifest Parameters.md

```diff
@@ -331,6 +280,27 @@ The POTA manifest is used to perform both a FOTA and SOTA update at the same tim
 
 ### Docker manifest examples
 
+#### Example of docker application manifest
```

Comment:
Add a new section for Application update. (Looks like its under docker manifest example).
Also did Teja confirm on the file extension type? May want to wait for a confirmation from her or Validation team.

### Application update manifest examples
#### Example of Debian package application update manifest
....

Also please change the level heading level for Docker compose
From #### Docker-Compose Manifest Examples
To ### Docker-Compose Manifest Examples

Reply:
done